server/locale/DE/GPL/styles/flex/ratio--20x1.css

```
@@ -0,0 +1,64 @@
+@media (min-aspect-ratio: 200/11) {
+    .message__headline {
+        font-size: 1.8vw;
+    }
+    .message__disclaimer {
+        font-size: 1.25vw;
+        line-height: 1.43rem;
+    }
+    .message__promo-container {
+        width: unset;
+    }
+    .message__logo {
+        max-width: 128px;
+    }
+    .message__logo:nth-of-type(1) {
+        width: 18%;
+    }
+    .message__logo:nth-of-type(2) {
+        width: 60%;
+        margin: 0 4px;
```

Changing the width of the logo and getting rid of the margin spaces it a little better.
```suggestion
        width: 55%;
```